modules.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, lib, config, ...}:
+let
+  toBG = { red, green, blue }:
+    "${toString(red   / 255.0)}, "   +
+    "${toString(green / 255.0)}, " +
+    "${toString(blue  / 255.0)})";
```

Comment:
this should be `"${toString(blue  / 255.0)}";` (no `)` at the end).

FWIW when we're already doing some magic I'd rather have a `backgroundcolor` value which is hex like `#FFFFFF` and then convert that. Something like [this](https://github.com/bertof/nix-rice/blob/main/color.nix#L40C1-L40C94) might work. 🤔 
If you don't feel like tinkering and just remove the `)`, that's also fine though

Reply:
Liked the idea of using nix-rice

I will fix it for now and add rice the next PR
Thanks for that

Reply:
```suggestion
    "${toString(blue  / 255.0)}";
```

Reply:
Options:
- Copy and paste from nix-rice
- Import it with flake (will work only in flakes)
- Import it without flake (set hash for nix-rice in the module)
- Other?

Reply:
Given we only need a single line (AFAIU) and it's MIT as well I'd just copypaste and prepend with a comment `# thanks to https://github.com/bertof/nix-rice`. :)

Reply:
@Melkor333  see  #10